qcodes_contrib_drivers/drivers/RohdeSchwarz/SMW200A.py

```diff
@@ -0,0 +1,2313 @@
+class AmplitudeModulation(InstrumentChannel):
+
+    def __init__(self, parent: 'RohdeSchwarz_SMW200A', name: str, hwchan: int, chnum: int):
+        """Combines all the parameters concerning the amplitude modulation. Activation
+        of amplitude modulation deactivates ARB, I/Q modulation, digital modulation
+        and all digital standards.
+
+        Args:
+            parent: the parent instrument of this channel
+            name  : the internal QCoDeS name of this channel
+            hwchan: the internal number of the hardware channel used in the communication
+            chnum : the internal number of the channel used in the communication
+
+        Attributes:
+            state: actives/deactivates the amplitude modulation. Values are 'ON' and 'OFF'.
+            source: Selects the modulation source. Values are:
+                'INT': internally generated LF signal = 'LF1' (channel 2 only with option SMW-K24)
+                'EXT': externally supplied LF signal  = 'EXT1' (channel 2 only with option SMW-K24)
+                'LF1': first internally generated signal
+                'LF2': second internally gererated signal (only available with option SMW-K24)
+                'NOIS': internally generated noise signal (only available with option SMW-K24)
+                'EXT1': first externally supplied signal
+                'EXT2': second externally supplied signal
+            depth: Sets the depth of the amplitude modulation in percent.
+            total_depth: Sets the total depth of the LF signal when using combined
+                signal sources in amplitude modulation.
+            coupling_mode: Selects the coupling mode. The coupling mode parameter also
+                determines the modefor fixing the total depth. Values are:
+                'UNC': Does not couple the LF signals. The deviation depth values of
+                both paths are independent.
+                'TOT': Couples the deviation depth of both paths.
+                'RAT': Couples the deviation depth ratio of both paths.
+            deviation_ratio: Sets the deviation ratio (path2 to path1) in percent.
+            sensitivity: (ReadOnly) Queries the sensitivity of the externally applied signal.
+                The sensitivity depends on the set modulation depth. The returned value
+                reports the sensitivity in %/V. It is assigned to the voltage value for
+                full modulation of the input.
+        """
+        self.hwchan = hwchan
+        self.chnum = chnum
+        super().__init__(parent, name)
+
+        self.add_parameter('state',
+                           label='State',
+                           set_cmd='SOUR{}:AM{}:'.format(self.hwchan, self.chnum)+'STAT {}',
+                           get_cmd='SOUR{}:AM{}:STAT?'.format(self.hwchan, self.chnum),
+                           val_mapping={'ON': 1, 'OFF': 0},
+                           vals=vals.Enum('ON', 'OFF'),
+                           docstring="""
```

Comment:
We do not need to have empty docstrings if there is nothing to write.

Reply:
This docstring doesn't look empty to me.